src/NUnitFramework/framework/Internal/Tests/Test.cs

```diff
@@ -188,6 +188,14 @@ public virtual string MethodName
         }
 
         /// <summary>
+        /// The arguments to use in creating the test or null if none required.
+        /// </summary>
+        public virtual object[] Arguments
+        {
+            get { return null; }
```

Comment:
Why not object[0]? Alternatively, why not just make this abstract since you end up overriding it anyway?